examples/heli.mapping

```diff
@@ -0,0 +1,12 @@
+emma_EastPE:aglaope_EastPE
```

Comment:
Remove this file and use `heliconius_mapping.mapping` or rename that file to this.

Reply:
Removed.